nilrt_snac/_verify.py

```diff
@@ -0,0 +1,15 @@
+import argparse
+
+from nilrt_snac._Configs import Configs
+
+from nilrt_snac import logger
+
+
+def _verify(args: argparse.Namespace) -> int:
+    """Configure SNAC mode."""
+    logger.info("Validating SNAC mode.")
```

Comment:
Is a mix of prints and logger output acceptable? If it is, then I am fine with changing this.

Reply:
Also, I used the same style as the bash script. I.e. `log INFO "Message"` became `logger.info("Message")`

Reply:
Yeah; mixing stdout and stderr is fine. Though you should know (because it's a linux secret) that they are not guaranteed to be well-ordered between each other. So you might have a few lines of stdout print before some stderr lines that were really emitted between each other.

Reply:
Everything that was `INFO` is now a print.